pkg/scheduler/schedule_one.go

```diff
@@ -594,6 +594,7 @@ func (sched *Scheduler) findNodesThatPassFilters(
 		for i := range feasibleNodes {
 			feasibleNodes[i] = nodes[(sched.nextStartNodeIndex+i)%numAllNodes]
 		}
+		diagnosis.EvaluatedNodes = len(feasibleNodes)
```

Comment:
This is same as the old logic(before #119779), however it seems a little weird to me.
As we don't actually `evaluate` some pods here(there is no filter plugin), I think evaluatedNodes equal to 0 is more reasonable to me.
WDYT @alculquicondor @sanposhiho